# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: dom/indexedDB/PBackgroundIDBCursor.ipdl
# Commit: f1be81af7d6b
# Full Hash: f1be81af7d6b1dd98040b90ff4e094da950f01f6
# Author: Simon Giesecke <sgiesecke@mozilla.com>
# Date: 2019-11-05 21:49:23
# Regressor Bug: 1168606
# File Overlap Count: 1
# Description:
#   Bug 1168606 - Support preloading also for index cursors. r=ttung,asuth
#   
#   Depends on D43252
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D43461
# ==============================================================================

diff -r 8dc185488801 -r f1be81af7d6b dom/indexedDB/PBackgroundIDBCursor.ipdl
--- a/dom/indexedDB/PBackgroundIDBCursor.ipdl	Tue Nov 05 12:48:36 2019 +0000
+++ b/dom/indexedDB/PBackgroundIDBCursor.ipdl	Tue Nov 05 12:45:48 2019 +0000
@@ -80,7 +80,7 @@
   nsresult;
   ObjectStoreCursorResponse[];
   ObjectStoreKeyCursorResponse;
-  IndexCursorResponse;
+  IndexCursorResponse[];
   IndexKeyCursorResponse;
 };
 
@@ -91,7 +91,8 @@
 parent:
   async DeleteMe();
 
-  async Continue(CursorRequestParams params, Key currentKey);
+  async Continue(CursorRequestParams params, Key currentKey,
+                 Key currentObjectStoreKey);
 
 child:
   async __delete__();
